knesset_data/dataservice/bills.py

```diff
@@ -0,0 +1,39 @@
+from base import (
+    BaseKnessetDataServiceCollectionObject, BaseKnessetDataServiceFunctionObject,
+    KnessetDataServiceSimpleField, KnessetDataServiceLambdaField
+)
+import constants
+
+
+class Bill(BaseKnessetDataServiceCollectionObject):
+    SERVICE_NAME = "api"
+    METHOD_NAME = "KNS_Bill"
+    ORDERED_FIELDS = [
+        ("id", KnessetDataServiceSimpleField('BillID', "integer", "the primary key")),
+        ("kns_num", KnessetDataServiceSimpleField('KnessetNum', "integer", "kneset number")),
+        ("name", KnessetDataServiceSimpleField('Name', "string", "bill heb name")),
+        ("type_id", KnessetDataServiceSimpleField('SubTypeID', "integer", "type id of the bill")),
+        ("type_description",
+         KnessetDataServiceSimpleField('SubTypeDesc', "string", "type description of the bill")),
+        ("private_num", KnessetDataServiceSimpleField('PrivateNumber', "integer",)),
+        ("committee_id", KnessetDataServiceSimpleField('CommitteeID', "integer",)),
```

Comment:
maybe it's worth to create a new class for this type of fields
something like:
("committee", KnessetdataServiceRelatedField("CommitteeID", dataservice_class=Committee))
which will handle it automatically
but need to make sure by default it doesn't fetch the related object - only when user requests it
so I'm not sure what's the best way to implement it..